### ISSUE TITLE
RichTextLabel: Wrong meta link detection when used inside a fill tag

### ISSUE THREAD

**Issue by timothyqiu** [godotengine/godot#35040]
**Godot version:** v3.2.beta.custom_build.b563de702

**OS/device including version:** macOS Catalina 10.15.2 / MacBook Pro (Retina, 13-inch, Early 2015)

**Issue description:**
The `[fill]` tag in RichTextLabel will adjust the word spacing to make lines to fill all the width of the control.

When the word spacing inside a link is adjusted by this tag, the visual display of words is correct, but the mouse cursor and signal emission acts as if the word spacing is not adjusted.

![demo](https://user-images.githubusercontent.com/372476/72215760-5356a880-3552-11ea-976c-4e5181cc47bb.gif)

This seems to be a long existing issue. I can reproduce this in 3.1.2.

**Steps to reproduce:**
1. Add a RichTextLabel to the scene
2. Set the BBCode to something like "[fill]This is a test and [url=test]here is alinkwith many words[/url] that wraps[/fill]"
3. Resize the RichTextLabel, so the word spacing in the link are adjusted
4. Run the project and try to find the link with your mouse

**Minimal reproduction project:** [RichTextLabelTest.zip](https://github.com/godotengine/godot/files/4050313/RichTextLabelTest.zip)

**Comment by KoBeWi** [comment]
Still valid in 3.2.4 beta3

**Comment by Bethleem** [comment]
I was going to report this same bug. It runs from at least from 3.1.1 to 3.2.4 beta. The issue is that the text behaves with the fill tag but the url don´t, it just always marks as interactive the areas where the text should be with no fill tag, no matter if fill is activated or not.

**Comment by Bethleem** [comment]
Indeed, It runs at least from the whole 3.0x series. In this [proyect](https://github.com/Bethleem/fill_bug/blob/main/bug%20url.zip) from 3.2.3 you can see two identical lines whit some last words as url. 
The first line has no fill tag and the url returns "1" int the editor when clicked. The second has the fill tag and a additional word for the fill kick in, and the url returns "2" in the editor when clicked. The interactive area of the second (fill) line is exactly the same as the first (no fill) line, despite it not matches the actual url words. You can see it clearly when click in the "el" word before the underline text of the second line and the absence of return in the final "ce" letters of the "enlace" word.

**Comment by timothyqiu** [comment]
This has been fixed on master by Complex Text Layouts, but still reproducible on 3.x.
